sequencer/api/espresso_dev_node.toml

```diff
@@ -0,0 +1,29 @@
+[meta]
+NAME = "espresso_dev_node_server"
+DESCRIPTION = "A server for debugging and developing with the espresso dev node"
+FORMAT_VERSION = "0.1.0"
+
+[route.devinfo]
+PATH = ["/dev-info"]
+DOC = """
+Get the debug info
+"""
+
+[route.freeze]
```

Comment:
Do we need the extra indirection by introducing new terminology "freeze/unfreeze" or should we just call it set-hotshot-down/up?

Reply:
Yeah makes sense. Will use `set-hotshot-down/up`

Reply:
e92c71737d74e7e3db24e017de9bf791f5bfaaac